test_runner/README.md

```diff
@@ -28,38 +28,37 @@ run the tests from within the `test_runner` directory.
 Test state (postgres data, pageserver state, and log files) will
 be stored under a directory `test_output`.
 
-You can run all the tests with:
-
-`pipenv run pytest`
-
-If you want to run all the tests in a particular file:
-
-`pipenv run pytest test_pgbench.py`
-
-If you want to run all tests that have the string "bench" in their names:
-
-`pipenv run pytest -k bench`
+* You can run all the tests with:
+  ```bash
+  pipenv run pytest
+  ```
```

Comment:
I'd also add an example for running tests in parallel (with `-n4` for example), to execute them faster.